doc/changes/latest.inc

```diff
@@ -353,7 +351,7 @@ Bug
 
 - Fix bug with :func:`mne.events_from_annotations(raw.annotations) <mne.events_from_annotations>` when ``orig_time`` of annotations is None and ``raw.first_time > 0``, by `Alex Gramfort`_
 
-- Fix bug in :func:`mne.set_eeg_reference` and related functions to set ``info['custom_ref_applied']`` to ``True`` for 'ecog' and 'seeg' channels in addition to 'eeg' by `Liberty Hamilton`_
```

Comment:
This removal also looks like a rebase error

Reply:
Still new at this.  Using vimdiff (meld not supported on Mac?), and probably misinterpreted a squiggly colored line.  I'll try once more, comparing directly to the current lines in master.